docs/static/jvm.asciidoc

```diff
@@ -49,7 +49,8 @@ Java HotSpot(TM) 64-Bit Server VM 18.9 (build 11.0.1+13-LTS, mixed mode)
 [[java-home]]
 ==== `LS_JAVA_HOME`
 
-{ls} uses the Java version set in `LS_JAVA_HOME`. The `LS_JAVA_HOME` environment
+{ls} uses the Java version set in `LS_JAVA_HOME`. If you prefer to use custom 
+JDK other than the version that is bundled, then the `LS_JAVA_HOME` environment
 variable must be set for {ls} to operate correctly.
```

Comment:
```suggestion
Logstash includes a bundled JDK which has been verified to work with each
specific version of Logstash, and generally provides the best performance and
reliability. 
If you need to use a JDK other than the bundled version, then set the
`LS_JAVA_HOME` environment variable to the version you want to use.
```

Reply:
Thank you for making this fix, @RobertShan2000!  This issue definitely needed clarification. Thank you for making it happen. 
Please consider updating the existing first sentence to emphasize the benefits of sticking to the bundled version a bit more (as indicated in the suggestion). We want people to have the option to set `LS_JAVA_HOME`, but we recommend that people stick to the bundled JDK unless they have a compelling reason for using another one.

Reply:
> 

@karenzone I refine the sentence as your suggestion, please let me know if you have any questions.